src/libraries/System.Net.Http/src/System/Net/Http/HttpRequestException.cs

```diff
@@ -28,6 +28,14 @@ public HttpRequestException(string message, Exception inner)
             }
         }
 
+        public HttpRequestException(string message, Exception inner, HttpStatusCode? statusCode)
```

Comment:
Nit: I know the other ctors don't currently have it, but please add XML comment documentation for this.  You can base it off of what's currently on the docs site for the other ctors, and just augment it with a param comment for statusCode.